lib/code_corps_web/controllers/category_controller.ex

```diff
@@ -16,7 +16,7 @@ defmodule CodeCorpsWeb.CategoryController do
   @spec show(Conn.t, map) :: Conn.t
   def show(%Conn{} = conn, %{"id" => id}) do
     with %Category{} = category <- Category |> Repo.get(id) |> preload() do
-      conn |> render("show.json-api", data: category)
+      conn |> render(CodeCorpsWeb.CategoryView, "show.json-api", %{ data: category, conn: conn, params: id })
```

Comment:
I wonder if we can just call the `show` action provided by the [jsonapi](https://github.com/jeregrine/jsonapi/blob/master/lib/jsonapi/view.ex#L106) lib instead of calling render....?

Reply:
Seems plausible. Have you tried this?